search-provider/eks-discovery-feed-provider.c

```diff
@@ -378,22 +378,13 @@ get_day_of_week (void)
   return g_date_time_get_day_of_week (datetime);
 }
 
+#define WRAPAROUND_CONSTANT 70
```

Comment:
I don't like hardcoding this either; indeed we should wrap around at the number of articles that are present in the database. I can't imagine finding the number to be very expensive, so I would try out the approach before dismissing it as not performing adequately.